src/service.py

```diff
@@ -161,20 +161,8 @@ def stop(self):
         print("Stopping Montagu...({}: {})".format(
             self.settings["instance_name"], self.settings["docker_prefix"]),
               flush=True)
-
-        # As documented in VIMC-805, the orderly container will
```

Comment:
I've moved this into https://vimc.myjetbrains.com/youtrack/issue/mrc-351 as we don't currently support it in orderly-web-deploy and it would be good to have